command/server/server.go

```diff
@@ -168,6 +168,16 @@ func setFlags(cmd *cobra.Command) {
 		),
 	)
 
+	cmd.Flags().Uint64Var(
+		&params.rawConfig.PubSubBuffersSize,
+		pubSubBufferSize,
+		defaultConfig.PubSubBuffersSize,
+		fmt.Sprintf(
+			"PubSub buffers size. Default: %ds",
```

Comment:
Please expand on the explanation for this flag. Also, please notice that the flag explanations are all lower-cased and follow a certain flag wording standard